### PR TITLE
[Begginer idea] CSSGram

### PR DESCRIPTION
**Pull request by AlexMayol**
As stated on the added file, the goal is to create a tool to add various filters to an uploaded image.

### CODE REVIEW

**Review by laisfrigerio — APPROVED** [review]

File: Projects/1-Beginner/CSSGram.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Product Landing Page
```
Comment (laisfrigerio) [inline Projects/1-Beginner/CSSGram.md]: Hey @AlexMayol  
Please, edit the name of exercise.

Comment (laisfrigerio) [inline Projects/1-Beginner/CSSGram.md]: And add this exercise reference on the README file

**Review by laisfrigerio — CHANGES_REQUESTED** [review]

**Comment by AlexMayol** [comment]
@laisfrigerio Nice catch, sorry! It's edited now 😄

**Comment by laisfrigerio** [comment]
Great job, @AlexMayol 

Thanks for your contribution!!!

**Pull request merged**